src/psyclone/domain/lfric/kernel_interface.py

```diff
@@ -82,14 +80,31 @@ class KernelInterface(ArgOrdering):
     TBD: This class should replace the current kernel stub generation
     code when all of its methods are implemented, see issue #928.
 
-    :param kern: the kernel that requires its kernel arguments to be \
-        created.
+    :param kern: the kernel for which to create arguments.
     :type kern: :py:class:`psyclone.dynamo0p3.DynKern`
 
     '''
+    field_mapping = {
```

Comment:
Thanks for moving these here. Please (briefly) document each of them with a `#:` comment (the colon means it gets picked up by Sphinx/autoapi).

Reply:
Well I never knew that. Added.